packages/pnpm-sync-lib/src/interfaces.ts

```diff
@@ -15,6 +15,67 @@ export interface ITargetFolder {
   folderPath: string;
 }
 
+/**
+ * @beta
+ */
+export enum LogMessageKind {
+  INFO = 'info',
+  WARNING = 'warning',
+  ERROR = 'error',
+  VERBOSE = 'verbose'
+}
+
+/**
+ * @beta
+ */
+export enum LogMessageIdentifier {
+  PREPARE_STARTING = 'Starting pnpm-sync prepare',
+  PREPARE_PROCESSING = 'Processing pnpm-sync prepare',
+  PREPARE_FINISHING = 'Finishing pnpm-sync prepare',
+  COPY_STARTING = 'Starting pnpm-sync copy',
+  COPY_PROCESSING = 'Processing pnpm-sync copy',
+  COPY_FINISHING = 'Finishing pnpm-sync copy'
+}
```

Comment:
It would be better for these Id's to be identifiers (`PREPARE_STARTING = 'prepare-starting'`) so they can be efficiently compared, used without importing the enum, and so that the wording of the error message could be revised later without breaking the API contract.

Reply:
Good suggestion!